### PR TITLE
Fix missing info from /api/uid/tsmeta - refs #498

### PR DESCRIPTION
**Pull request by lburg**
Unfortunately this breaks the [parseFromColumnWithUIDMetaNSU](https://github.com/OpenTSDB/opentsdb/blob/next/test/meta/TestTSMeta.java#L400) test as the `NoSuchUniqueId` exception is not raised anymore since `LoadUIDs.call` [returns early](https://github.com/OpenTSDB/opentsdb/blob/next/src/meta/TSMeta.java#L837) due to `meta` being `null`. I didn't know how to fix the test, whether to keep it as-is and try to raise the same exception again, or change it to make the tsuid `000001000001000002` a valid TSUID missing a UID meta object.

What's the best approach?

### CODE REVIEW

**Comment by manolama** [comment]
Nice catch, I'd follow the second approach for the unit test and make sure it throws the NSUI.

File: src/meta/TSMeta.java [hunk 1]
```diff
@@ -405,24 +405,21 @@ public ValidateCB(final TSMeta local_meta) {
     if (column.value() == null || column.value().length < 1) {
       throw new IllegalArgumentException("Empty column value");
     }
-    
-    final TSMeta meta = JSON.parseToObject(column.value(), TSMeta.class);
+
+    final TSMeta parsedMeta = JSON.parseToObject(column.value(), TSMeta.class);
```
Comment (manolama) [inline src/meta/TSMeta.java]: Minor nitpick, ``parsed_meta`` :)

Comment (lburg) [inline src/meta/TSMeta.java]: Oops, I've been tricked by the methods' names!

**Comment by lburg** [comment]
After reading the `before` method more carefully, I managed to fix the test. Let me know if it was done correctly!

**Comment by manolama** [comment]
Merged in b1c32620cd008572fcf9e7ed8c9701c22eac85b2

**Pull request closed without merging**